examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -519,7 +519,7 @@ export const TaskListInstantiationFactory = new DataObjectFactory<TaskList>(
  */
 export class BaseDocument extends DataObject implements IBaseDocument {
 	private readonly taskListCollection = new Map<string, TaskList>();
-
+	private readonly leader = "leader";
```

Comment:
This is always "leader"? I guess I'm not clear on what this value represents if it isn't tracking the current leader. Probably worth some documentation if it represents something else.